build_tools/benchmarks/common/benchmark_config_test.py

```diff
@@ -66,28 +68,87 @@ def test_build_from_args(self):
     self.assertEqual(config, expected_config)
 
   def test_build_from_args_benchmark_only(self):
-    args = build_common_argument_parser().parse_args([
-        f"--tmp_dir={self.tmp_dir.name}",
+    args = common_arguments.build_common_argument_parser().parse_args([
+        f"--tmp_dir={self.tmp_dir_path}",
         f"--normal_benchmark_tool_dir={self.normal_tool_dir}",
-        self.build_dir.name
+        str(self.build_dir_path)
     ])
 
-    config = BenchmarkConfig.build_from_args(args=args, git_commit_hash="abcd")
+    config = benchmark_config.BenchmarkConfig.build_from_args(
+        args=args, git_commit_hash="abcd")
 
     self.assertIsNone(config.trace_capture_config)
 
   def test_build_from_args_invalid_capture_args(self):
-    args = build_common_argument_parser().parse_args([
-        f"--tmp_dir={self.tmp_dir.name}",
+    args = common_arguments.build_common_argument_parser().parse_args([
```

Comment:
Seems like this name could be shorter if it's going to be namespaced:

```suggestion
    args = common_arguments.parser().parse_args([
```

Reply:
Done.